rc/tools/doc.kak

```diff
@@ -137,9 +137,19 @@ define-command -params 1 -hidden doc-render %{
 define-command -params 1..2 \
     -shell-script-candidates %{
         if [ "$kak_token_to_complete" -eq 0 ]; then
-            find "${kak_runtime}/doc/" -type f -name "*.asciidoc" | sed 's,.*/,,; s/\.[^/]*$//'
+            (
+                find "${kak_runtime}/doc/" -type f -name "*.asciidoc"
+                find "${kak_runtime}/rc/" -type f -name "*.asciidoc"
+                find "${kak_config}/autoload/" -type f -name "*.asciidoc"
```

Comment:
I wonder if the same order is guaranteed by
`find "${kak_runtime}/doc/" "${kak_runtime}/rc/" "${kak_config}/autoload/" -type f -name "*.asciidoc"`

Reply:
I did not realise that was a thing `find(1)` could do! Thanks!